src/components/icons-with-counter/IconsWithCounter.tsx

```diff
@@ -0,0 +1,45 @@
+import { styles } from '~/components/icons-with-counter/IconsWithCounter.style'
+import { useState, useEffect } from 'react'
+
+import { Box } from '@mui/system'
```

Comment:
we use all mui imports from '@mui/material' and it should be like this: import Box from '@mui/material/Box', it will reduce the bundle size.

Also organize your imports: libs - component,containers,hooks - utils,const,styles,types

Reply:
thank`s , done